docs/index.html

```diff
@@ -15,20 +15,6 @@
         <div class='py1 px2'>
```

Comment:
Please discard the changes to this file, this should only be modified on a release commit.